crates/l2/prover/sp1_prover.rs

```diff
@@ -2,8 +2,7 @@ use tracing::info;
 
 use sp1_sdk::{ProverClient, SP1ProofWithPublicValues, SP1ProvingKey, SP1Stdin, SP1VerifyingKey};
 
-/// The ELF (executable and linkable format) file for the Succinct RISC-V zkVM.
-pub const FIBONACCI_ELF: &[u8] = include_bytes!("./sp1/program/elf/riscv32im-succinct-zkvm-elf");
+use crate::utils::config::prover::ProverConfig;
 
 pub struct SP1Prover {
```

Comment:
I know this is not in this PR's scope, but let's rename this to `Prover`

Reply:
Also the modules to `prover.rs`

Reply:
Done